codebuild/bin/s2n_codebuild.sh

```diff
@@ -49,7 +49,7 @@ if [[ "$OS_NAME" == "linux" && "$TESTS" == "valgrind" ]]; then
     # and will not produce any output
     while sleep 9m; do echo "=====[ $SECONDS seconds still running ]====="; done &
 
-    if [[ "$S2N_LIBCRYPTO" == "openssl-1.1.1" || "$S2N_LIBCRYPTO" == "awslc" ]]; then
+    if [[ "$S2N_LIBCRYPTO" == "openssl-1.1.1" ]]; then
```

Comment:
Not sure I understand this part. It is currently passing in CI without those suppressions so I am wondering why you are seeing more leaks now?

Reply:
It is currently passing in CI because of the wildcard suppression. After removing the wildcard suppression, it would require adding [these](https://github.com/jmayclin/s2n-tls/blob/aws-lc-valgrind-suppressions/tests/unit/valgrind.suppressions) suppressions, which seemed too numerous to be reasonable.